load_site.py

```diff
@@ -0,0 +1,27 @@
+import asyncio
+import aiohttp
+import time
+
+
+async def load_yandex():
+    with aiohttp.ClientSession() as s:
+        resp = await s.get('https://ya.ru')
+        print(await resp.json())
+
+
+async def load_google():
+    with aiohttp.ClientSession() as s:
+        resp = await s.get('https://google.com')
+        print(await resp.json())
```

Comment:
здесь очевидно дублирование кода
предложу как одной функцией с параметром:

```suggestion
async def load_resource(resource):
    async with aiohttp.ClientSession() as s:
        resp = await s.get(resource)
        print(await resp.text())
```

а в момент вызова передавать необходимый ресурс